admin-perspective/olm/install-forcenamespace/install-forcenamespace.md

```diff
@@ -0,0 +1,85 @@
+
+![Operator namespace not defined single](img/1-7-noDefinedNS-single.png)
+- Changing the availability namespace no longer has any affect on the install namespace, as it has been manually changed.
+
+## Installing with an operator-defined namespace
+
+![Operator namespace defined all](img/2-1-definedNS-all.png)
+- When an operator-defined installation namespace exists, that namespace will be the default install namespace for the operator.
+- The operator-defined namespace can be used for **all namespace** availability, or **specific namespace** availability, as the operator's InstallModes allow.
+- If the user viewed the **Advanced Installed Namespace Options** while **all namespaces** was selected, the only other namespace available for selection besides the operator-defined namespace would be **openshift-operators**.
+
+![Operator namespace defined single](img/2-2-definedNS-single.png)
+- The availability of this operator is changed to **specific namespace** and the operator-defined namespace is used as the default.
+- Changing the availability namespace in this case would not affect the install namespace as it is the operator-defined namespace. The only way the install namespace would be changed is if the user manually changes it.
+
+![Operator namespace defined single](img/2-3-definedNS-single.png)
+- The user expands the **Advanced Installed Namespace Options** and can select a different namespace other than the operator-defined.
+
+![Operator namespace defined single](img/2-4-definedNS-single.png)
+- The user selects another install namespace.
+
+![Operator namespace defined single](img/2-5-definedNS-single.png)
+- This operator will now be installed in namespace **abc** and available to namespace **desiredNamespace**.
+
+## When an operator-defined namespace doesn’t exist
+
+![Operator namespace defined doesn't exist all](img/3-1-definedNSCreate-all.png)
+- When an operator-defined namespace doesn't exist, the user is notified via an inline alert that it will be created when the operator is installed.
+- The **Show/Hide Advanced Installed Namespace Options** controls would not appear in this case so that the user is always aware of the additional alert.
+
+![Operator namespace defined doesn't exist single](img/3-2-definedNSCreate-single.png)
+- **Specific namespace** option is also supported.
+
+## Enabling cluster monitoring on a namespace
+
+![Operator namespace defined monitor all](img/4-1-definedNSMonitor-all.png)
+- Operators can also request cluster monitoring be enabled on the namespaces they are installed on.
+- This operator has a defined namespace of **openshift-desiredNamespace** and is requesting cluster monitoring be enabled on it.
+- The **Show/Hide Advanced Installed Namespace Options** controls would not appear in this case so that the user is always aware of the additional option.
```

Comment:
It's a bit confusing that in this case we are not showing the link at all.

Reply:
We need the user to always see the additional "Enable cluster monitoring" option so the thought was we'd forego the show/hide behavior in this case (and when the namespace is being created.)

Reply:
right but in this case we are not hiding the part of the form, which actually is a bit complex.
Would rather have it uniform and dont add any additional functionality if it's not necessary.